### PR TITLE
 Added error reporting to session code

### PR DESCRIPTION
**Pull request by dyeldandi**
This is a replacement for pull request #165

### CODE REVIEW

File: php_memcached_session.c [hunk 1]
```diff
@@ -442,6 +442,7 @@ PS_READ_FUNC(memcached)
 		*val = ZSTR_EMPTY_ALLOC();
 		return SUCCESS;
 	} else {
+		php_error_docref(NULL TSRMLS_CC, E_WARNING, "error getting session from memcached: %s", memcached_last_error_message(memc_sess->memc_sess));
```
Comment (sodabrew) [inline php_memcached_session.c]: I think this can be `memc` instead of `memc_sess->memc_sess`

**Comment by sodabrew** [comment]
```
php_memcached_session.c:475:118: error: ‘memc_sess’ undeclared (first use in this function)
    php_error_docref(NULL TSRMLS_CC, E_WARNING, "error saving session to memcached: %s", memcached_last_error_message(memc_sess->memc_sess));
                                  
```

**Comment by dyeldandi** [comment]
Ah, yes, you are correct. Fixed.

**Comment by sodabrew** [comment]
Thanks!

**Pull request merged**